package.json

```diff
@@ -1,15 +1,23 @@
 {
   "name": "webpush-client",
-  "version": "0.4.2",
-  "description": "My New Project description.",
-  "main": "index.js",
-  "repository": {
-    "url": "https://github.com/bpolaszek/webpush-js.git",
-    "type": "git"
+  "version": "0.5.8",
+  "description": "Web push client",
+  "main": "dist/index.min.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc && npm run copy && npm run minify",
+    "copy": "cpy src/webpush-sw.js dist",
+    "minify": "terser -c -m -o dist/index.min.js -- dist/index.js && terser -c -m -o dist/webpush-client.min.js -- dist/webpush-client.js"
   },
-  "author": "Beno!t POLASZEK <bpolaszek@gmail.com>",
-  "license": "MIT",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
   "devDependencies": {
-    "@symfony/webpack-encore": "^0.21.0"
-  }
-}
+    "cpy-cli": "^3.1.1",
```

Comment:
Why this dependency?

Reply:
I am using this to copy the file webpush-sw.js to dist folder during `build` script 

```
  "scripts": {
    "build": "tsc && npm run copy && npm run minify",
    "copy": "cpy src/webpush-sw.js dist",
 ```